src/covizpy/plot_summary.py

```diff
@@ -22,9 +30,65 @@ def plot_summary(
     fun : str, optional
         Aggregation function for val, by default 'sum'
     date_from : str, optional
-        Start date for plot summary, by default '2022-01-01'
+        Start date of the data range with format '%Y-%m-%d'. By default 'None' is used to represent 7 days prior to today's date
     date_to : str, optional
-        End date for plot summary, by default '2022-01-13'
+        End date of data range with format '%Y-%m-%d'. By default 'None' is used to represent today's date
```

Comment:
```suggestion
        End date of data range with format 'YYYY-MM-DD'. By default 'None' is used to represent today's date
```
Align to use above for better understanding.